lib/SIL/SILOwnershipVerifier.cpp

```diff
@@ -879,11 +878,11 @@ void SILInstruction::verifyOperandOwnership() const {
   if (isa<TermInst>(this))
     return;
 
-  ErrorBehaviorKind errorBehavior;
+  LinearLifetimeChecker::ErrorBehaviorKind errorBehavior;
   if (IsSILOwnershipVerifierTestingEnabled) {
-    errorBehavior = ErrorBehaviorKind::PrintMessageAndReturnFalse;
+    errorBehavior = decltype(errorBehavior)::PrintMessageAndReturnFalse;
```

Comment:
I'm surprised you needed to use `decltype` here.

Reply:
I was just trying to avoid an 80+ line error. I found the extra :: made it hard.